packages/lms/src/routes/+layout.svelte

```diff
@@ -34,7 +58,7 @@
   </svelte:fragment>
   <svelte:fragment slot="sidebarRight">
     <div class="p-2">
-      <EntityNav entities={data.sitemap} />
+      <EntityNav entities={data.sitemap} on:click={handleNavItemClick} />
```

Comment:
I am not 100% sure where it's from and it might be my project set up wrong as your video works fine
I have 4 main topics in my content folder - and no matter what element i click of my top level elements everything expands or collapses. 

Can someone confirm / disproof my experience here?

Reply:
Yeah I get the same the onclick is basically applying to all items in the top level of the menu, and as such opening and closing them all.

Reply:
@Anclagen, could you please take a look at the recent changes in the navigation menu?  Do you have any suggestions code  on how to fix this issue?

Reply:
This will fix the problem of all top level items opening at once.
```suggestion
    {#each data.sitemap as entity}
      <EntityNav entities={entity} />
    {/each}
```